tests/src/CoreMangLib/system/span/BasicSpanTest.cs

```diff
@@ -0,0 +1,672 @@
+using System;
```

Comment:
@AtsushiKan, it would be great if fast and slow span tests were equivalent to ensure the same semantics.